ml-agents/mlagents/trainers/tf_policy.py

```diff
@@ -89,18 +89,39 @@ def _initialize_graph(self):
             init = tf.global_variables_initializer()
             self.sess.run(init)
 
-    def _load_graph(self):
+    def load_or_finetune_graph(self, trainer_parameters: Dict[str, Any]) -> None:
+        """
+        Loads the graph either fresh, or in finetune mode.
+        """
+        if trainer_parameters["finetune_path"]:
```

Comment:
Is `finetune_path` always set? I'd probably do this as
```
finetune = "finetune_path" in trainer_parameters
model_load_path = trainer_parameters["finetune_path"] if finetune else self.model_path
self._load_graph(model_load_path)
```
and change _load_graph() to
```
def _load_graph(self, model_load_path: str = None) -> None:
```

Reply:
This is simpler, nice

Reply:
Sorry, didn't realize it was on a closed branch at the time. But I think the same rec applies.